client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -27,6 +28,12 @@ public class SchemaRegistryClientConfig {
   public static final String SCHEMA_REGISTRY_USER_INFO_CONFIG =
       "schema.registry.basic.auth.user.info";
   public static final String USER_INFO_CONFIG = "basic.auth.user.info";
+  public static final String SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS =
+      "srclient.http.connect.timeout.ms";
```

Comment:
```suggestion
      "http.connect.timeout.ms";
```

Reply:
@varunpv Sure, lgtm.

Reply:
Connect prepends `value.converter` or `key.converter`, so we should remove `srclient.`